presidio-image-redactor/Pipfile

```diff
@@ -6,11 +6,11 @@ name = "pypi"
 [packages]
 flask = ">=1.1.2"
 pytesseract = ">=0.3.7"
-presidio-analyzer = ">=2.2.0"
+presidio-analyzer = "==2.2.31"
```

Comment:
Do we want this fixed?

Reply:
When it wasn't, there was an issue with the latest spacy when starting the app in Docker, will give it another try now

Reply:
Fixed